### ISSUE TITLE
Accessability

### ISSUE THREAD

**Issue by dternyak** [MyCryptoHQ/MyCrypto#493]
Ensure that at a minimum, aria-label attributes are at parity with V3.

Notes:
![-2017-11-29 at 2 19 40 pm](https://user-images.githubusercontent.com/7861465/33402154-4b5bd2b0-d521-11e7-8b23-a10929ce25ec.png)

**Comment by tayvano** [comment]
Reference: http://simplyaccessible.com/article/react-a11y/

**Comment by wbobeirne** [comment]
I've got some experience with accessibility (contracts with required WCAG compliance, grueling stuff) and know how to use a screen reader, I'll definitely give this a once-over when we get closer to public beta. But of course, any additional eyes and ears are helpful!

**Comment by tayvano** [comment]
FYI:

> Oh not a problem at all, it is my pleasure. Happy to help with disability/readability issues anytime for free, you are welcome to @ me, dm me, or email me > honeypot (at) http://promether.com 

https://twitter.com/MadameHoneyPot/status/936002085870587904

**Comment by wbobeirne** [comment]
Did a pretty quick audit and ran into plenty of issues. Bold ones are high priority, italicized ones are not the most important. All in all though, the site seemed entirely accessible to me, with not too many pain points.

### Global Components
* **Modals and screen covers (Like the alpha agreement) need to use the [dialog attribute](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_dialog_role)**
* **Modal close button needs descriptive text**
* Footer social icons need descriptive text
* **More alerts should be using `aria-live`, `"assertive"` if it's very important and `"polite"` if it can wait.** Notifications do this, and it works very well.

### Generate Wallet
* Private key in disabled input has aria-label of [Object object]
* Print paper wallet page has multiple `h1` elements
* "Print Paper Wallet" button has aria-label of [Object object]

### Send Transaction
* Online / Offline image needs alt text
* Promo images need alt text
* Promo buttons need descriptive text
* Token row balance title attribute is in wei, very hard to understand, and says to double click, which doesn't do anything on this element
* Close wallet selector X button needs descriptive text
* _Generate transaction button should change to indicate it's loading, there's no feedback to the screen reader_

### Swap
* Bity logo needs alt text
* Multiple h1 elements

**Comment by dternyak** [comment]
It may be wise to creator a tslint plugin for ensuring that all text is rendered via `translation`. We could also extend the plugin to make sure that images have alt-text. It may be worth having a small brainstorming session on the various checks we could have CI perform to ensure that we're maintaining accessibility as we add new features.

**Comment by dternyak** [comment]
##### We can use this list to brainstorm:

- Image alt-text
- All text rendered by `transaction` (no raw english text in JSX)
